src/OrleansZooKeeperUtils/project.json

```diff
@@ -0,0 +1,19 @@
+﻿{
+  "version": "1.0.0-*",
+  "description": "OrleansZooKeeperUtils Class Library",
+  "authors": [ "Gutemberg" ],
```

Comment:
Its the default from the template but don't worry, before we merge we will add those details since it is used to generate the output nugets. ;)

Sent from my iPhone

> On 24 de mar de 2016, at 02:52, Gabriel Kliot <notifications@github.com> wrote:
> 
> In src/OrleansZooKeeperUtils/project.json:
> 
> > @@ -0,0 +1,19 @@
> > +﻿{
> > +  "version": "1.0.0-*",
> > +  "description": "OrleansZooKeeperUtils Class Library",
> > +  "authors": [ "Gutemberg" ],
> Gutemberg is the author everywhere! ;-)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>